solution-jivanov/dynamodb.yml

```diff
@@ -0,0 +1,15 @@
+Resources:
+
+  UsersTable:
+    Type: AWS::DynamoDB::Table
```

Comment:
If we need to front this Dynamo table with read/write-through cache layer for scaling needs, how would you approach that?

Reply:
DAX is probably a good choice. And it can be provisioned via CloudFormation too i.e. it is fairly straightforward IaC as well.